src/ui/widgets/common/WidgetForm.js

```diff
@@ -25,6 +25,15 @@ export const MODE_CLONE = 'clone';
 const maxLength30 = maxLength(30);
 const maxLength70 = maxLength(70);
 
+export const validateFieldValueForFormikValidators = (value, validators) => {
+  let error = null;
+  validators.forEach((validator) => {
+    const validationError = validator(value);
+    if (validationError) error = validationError;
+  });
+  return error;
+};
+
```

Comment:
here you can replace the function with the utility function that we have in formikUtils